server/data/healthcheck.js

```diff
@@ -1,4 +1,5 @@
 const superagent = require('superagent')
+/** @type {any} */
```

Comment:
There's a bug in the typescript definitions here - it doesn't recognise Agent as a constructor: https://github.com/node-modules/agentkeepalive/issues/76